lib/browser.js

```diff
@@ -117,7 +118,16 @@ class Browser {
   }
 
   async _launch (preview) {
+    const isPkg = typeof process.pkg !== 'undefined'
```

Comment:
(nitpick): it should be moved as a top-level declaration (after `require` statements)

Reply:
Sure. I will do so.